keyboards/synthlabs/065/info.json

```diff
@@ -0,0 +1,432 @@
+{
+  "keyboard_name": "065",
+  "manufacturer": "Synth Labs",
+  "url": "http://065.synthlabs.io",
+  "maintainer": "hongaaronc",
+  "processor": "RP2040",
+  "bootloader": "rp2040",
+  "diode_direction": "COL2ROW",
+  "debounce": 5,
+  "features": {
+    "bootmagic": true,
+    "command": false,
+    "console": false,
+    "extrakey": true,
+    "mousekey": true,
+    "nkro": true,
+    "rgblight": false,
```

Comment:
```suggestion
```

Reply:
Whoops yeah that makes sense. Realize now that those values are default.